packages/common/src/types.ts

```diff
@@ -131,3 +131,13 @@ export enum BookingChange {
   BOOKED = "BOOKED",
   CANCELLED = "CANCELLED",
 }
+
+export type BookedDonationWithDonor = {
```

Comment:
BookedDonationWithDonorDetails

(We don't actually have the donor object here)